src/backend/metal/src/command.rs

```diff
@@ -1523,6 +1561,9 @@ where
         Cmd::BindPipeline(ref pipeline_state) => {
             encoder.set_render_pipeline_state(pipeline_state.borrow());
         }
+        Cmd::UseResource { resource, usage } => {
+            encoder.use_resource(resource.as_native(), usage);
```

Comment:
We'll need to update this soon because this is deprecated now https://developer.apple.com/documentation/metal/mtlrendercommandencoder/2866168-useresource

Reply:
Yeah, I saw that. Not exactly sure when the method we do call becomes unavailable though.